tests/unit_tests/blas/batch/gemm_batch_stride.cpp

```diff
@@ -48,12 +48,65 @@ extern std::vector<sycl::device *> devices;
 namespace {
 
 template <typename fp>
+typename std::enable_if<std::is_integral<fp>::value, bool>::type check_equal_int(fp x, fp x_ref,
+                                                                                 int error_mag) {
+    return (std::abs(x - x_ref) <= 1);
+}
+
+template <class T>
+struct acc_type {
+    typedef host_accessor<T, 1, access_mode::read> type;
+};
+
+template <class T>
+using acc_type_t = typename acc_type<T>::type;
```

Comment:
Is there a benefit of the `acc_type` struct over the suggestion below?
```
using acc_type_t = host_accessor<T, 1, access_mode::read>;
```
Same comment for `vec_type` below.